packages/nakama-js/socket.ts

```diff
@@ -1034,6 +1036,12 @@ export class DefaultSocket implements Socket {
       }
 
       if (this.verbose && window && window.console) {
+        if (untypedMessage.match_data_send) {
+          untypedMessage.match_data_send.data = decode(untypedMessage.match_data_send.data);
```

Comment:
this is still modifying the message that will actually get sent down the wire, won't it?

Reply:
Changed the code to copy `untypedMessage` and log that. Feel free to share any ideas :)